map_coloring.py

```diff
@@ -20,8 +20,8 @@
 import shapefile
 import matplotlib
 import networkx as nx
-from dimod import DiscreteQuadraticModel
-from dwave.system import LeapHybridDQMSampler
+from dimod import CQM, BinaryQuadraticModel
```

Comment:
Suggest using full name:
```suggestion
from dimod import ConstrainedQuadraticModel, BinaryQuadraticModel
```